frontend/src/redux/reducers/ui.tsx

```diff
@@ -65,14 +68,14 @@ function reducer(state = INITIAL_STATE, action: Action) {
       newFilters.sidebar = {
         ...newFilters.sidebar,
         selected: action.selected,
-        isVisible: !!action.selected,
+        isVisible: !!action.selected
```

Comment:
This is not needed and actually adds noise to the diff when we add a new line to this object. Same below.